src/main/java/com/ust/sourcecourse/configuration/request/TagDescriptionRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.ust.sourcecourse.configuration.request;
+
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class TagDescriptionRequest {
+
+	private String description;
+
+	private List<String> Tags;
```

Comment:
Follow Java naming conventions. Variable name should start with lower case letters. 
Change to tags and change everywhere accordingly.

Reply:
resolved